app.js

```diff
@@ -50,6 +50,10 @@ app.engine('hbs', hbs({
       } else {
         return 'fa fa-arrow-right fr'
       }
+    },
+    bgImg: (name) => {
+      name = name.toLowerCase().replace(/\s/g, '-')
```

Comment:
maybe you can remove toLowerCase() and use replace (/[a-z]) is the same toLoweCase